tensorboard/webapp/webapp_data_source/tb_server_data_source.ts

```diff
@@ -49,11 +50,23 @@ export class TBServerDataSource {
     return this.http.get<PluginsListing>(pathWithParams);
   }
 
-  fetchRuns() {
-    return from(this.tfBackend.runsStore.refresh());
+  fetchRuns(): Observable<Run[]> {
```

Comment:
Would be nice if this implementation was consistent with fetchEnvironment().  Feel free to refactor fetchEnvironment() to look more like this.

I do think some explanation of the implications of these two calls (like the one in fetchEnvironment()) is valuable.

Reply:
Done, made them look equal.